ConfigWindow.xaml.cs

```diff
@@ -111,8 +112,20 @@ private void UpdateResBox(bool isGif)
         /// <param name="e">Event args</param>
         private void ResolutionSelector_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            Config.Instance.CamWidth = Convert.ToInt32(((string)ResolutionSelector.SelectedItem).Split('x').First());
-            Config.Instance.CamHeight = Convert.ToInt32(((string)ResolutionSelector.SelectedItem).Split('x').Last());
+            try
+            {
+                Config.Instance.CamWidth = Convert.ToInt32(((string)ResolutionSelector.SelectedItem).Split('x').First());
+                Config.Instance.CamHeight = Convert.ToInt32(((string)ResolutionSelector.SelectedItem).Split('x').Last());
+            }
+            catch (Exception)
+            {
+                MessageBox.Show(
+                    this,
+                    "Unable to determine camera resolution of your selected camera.",
+                    "Camera resolution",
+                    MessageBoxButton.OK,
+                    MessageBoxImage.Warning);
```

Comment:
I think I'd prefer to log this instead of showing a message box
```suggestion
                Logger.Log("Unable to determine camera resolution of the selected camera", LogLevel.Warning);
```